### PR TITLE
php: add German translation

### PR DESCRIPTION
**Pull request by TheBulaDev**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

### CODE REVIEW

File: pages.de/common/php.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
+# php
+
+> PHP Befehlszeilenschnittstelle.
+> Mehr Informationen: <https://php.net>.
+
+- Analysieren Sie ein PHP-Skript und führen Sie es aus:
+
+`php {{datei}}`
```
Comment (kbdharun) [inline pages.de/common/php.md:8]:
```suggestion
`php {{pfad/zu/datei}}`
```

File: pages.de/common/php.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+- Überprüfen Sie die Syntax eines PHP-Skripts:
+
+`php -l {{datei}}`
```
Comment (kbdharun) [inline pages.de/common/php.md:12]:
```suggestion
`php -l {{pfad/zu/datei}}`
```

**Review by kbdharun — COMMENTED** [review]
Thanks for your contribution. I have some suggestions for this page.

The path placeholder seems to be only translated as `file` instead of `path/to/file`.

File: pages.de/common/php.md [hunk 3]
```diff
@@ -0,0 +1,36 @@
+# php
+
+> PHP Befehlszeilenschnittstelle.
+> Mehr Informationen: <https://php.net>.
+
+- Analysieren Sie ein PHP-Skript und führen Sie es aus:
+
+`php {{datei}}`
+
+- Überprüfen Sie die Syntax eines PHP-Skripts:
```
Comment (gutjuri) [inline pages.de/common/php.md:10]:
```suggestion
- Überprüfe die Syntax eines PHP-Skripts:
```

File: pages.de/common/php.md [hunk 4]
```diff
@@ -0,0 +1,36 @@
+# php
+
+> PHP Befehlszeilenschnittstelle.
+> Mehr Informationen: <https://php.net>.
+
+- Analysieren Sie ein PHP-Skript und führen Sie es aus:
```
Comment (gutjuri) [inline pages.de/common/php.md:6]:
```suggestion
- Analysiere ein PHP-Skript und führe es aus:
```

File: pages.de/common/php.md [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+
+`php -l {{datei}}`
+
+- Führen PHP interaktiv:
+
+`php -a`
+
+- Führen Sie einen PHP-Code aus:
+
+`php -r "{{code}}"`
```
Comment (gutjuri) [inline pages.de/common/php.md:20]:
```suggestion
`php -r "{{befehl}}"`
```

**Review by gutjuri — CHANGES_REQUESTED** [review]

File: pages.de/common/php.md [hunk 6]
```diff
@@ -0,0 +1,36 @@
...
+
+- Holen Sie sich eine Liste der installierten PHP-Erweiterungen:
+
+`php -m`
+
+- Informationen zur aktuellen PHP-Konfiguration anzeigen:
+
+`php -i`
+
+- Informationen zu einer bestimmten Funktion anzeigen:
```
[the remark this answers is not in the document]
Comment (TheBulaDev) [reply pages.de/common/php.md:34]: Vielen Dank für alle Ihre Vorschläge

**Review by gutjuri — APPROVED** [review]

**Review by kbdharun — APPROVED** [review]
LGTM, Thanks for your contribution.

**Pull request merged**